nbactions.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file for ?

Reply:
a, crap, that should not be there. Hold on let me fix